dist/common/scripts/scylla_cpuset_setup

```diff
@@ -13,7 +13,7 @@ import argparse
 from scylla_util import *
 
 if __name__ == '__main__':
-    if os.getuid() > 0:
+    if not is_container() and os.getuid() > 0:
```

Comment:
I don't understand this check. Suppose we're in a container and we're not root. This check allows us to continue. But we'll fail since we're not root. No?

Reply:
> I don't understand this check. Suppose we're in a container and we're not root. This check allows us to continue. But we'll fail since we're not root. No?

I forgot to describe that `is_nonroot()` does not checking current uid, is checks $SCYLLA_DIR/SCYLLA-NONROOT-FILE instead.
```
def is_nonroot():
    return Path(scylladir_p() / 'SCYLLA-NONROOT-FILE').exists()
```
We only create this file when we install by unified installer with --nonroot option.

The problem is, current nonroot mode hardcoded data directory == scylla root directory, as I described.

Reply:
Ok. Let's plan to improve this later.